app/model/editions/EditionsTemplates.scala

```diff
@@ -146,8 +146,12 @@ case class FrontTemplate(
   def swatch(swatch: Swatch) = copy(presentation = FrontPresentation(swatch))
 }
 
+case class TimeWindowConfigInDays(startDiff: Int, endDiff: Int)
+case class CapiQueryPrefillParams(timeWindowConfig: TimeWindowConfigInDays)
```

Comment:
Could use a default here to have a simpler interface:

```suggestion
case class CapiQueryPrefillParams(timeWindowConfig: TimeWindowConfigInDays = TimeWindowConfigInDays(0, 0))
```

Reply:
I had that at the beginning, but then i decided that it is more clear to specify those values in templets config so it will be clear which values are used by which edition
Default values can cause that we may forget that those configs exists